docs/index.md

```diff
@@ -0,0 +1,138 @@
+# uData
+
+[![Python Requirements Status][requires-io-badge]][requires-io-url]
+[![JavaScript Dependencies Status][david-dm-badge]][david-dm-url]
+[![JavaScript Development Dependencies Status][david-dm-dev-badge]][david-dm-dev-url]
+[![Join the chat at https://gitter.im/etalab/udata][gitter-badge]][gitter-url]
+
+
+_uData_ is an open and social data hub.
```

Comment:
I think that we should be more explicit on who is using it for what. But that can wait another PR :)